docs/content/modeling/organization-context-authorization.mdx

```diff
@@ -273,7 +278,7 @@ Note that so far, we have not prevented Anne from viewing "Project X" even if An
 
 ### Take Organization Context Into Consideration
 
-##### Extend The Authorization Model
+##### Extending The Authorization Model
```

Comment:
In this article, we are sometimes using present tense of a verb and other times the root verb in section titles. Let's settle on one

Reply:
Will change to present tense (i.e., Extend not extending) to keep things clean